stripper/ze_paranoid_rezurrection_csgo1.cfg

```diff
@@ -1,50 +1,310 @@
-; "ze_paranoid_rezurrection_csgo1" config, made by "Berke" "STEAM_1:0:95142811", version 1.
+// "ze_paranoid_rezurrection_csgo1" config, made by "Berke" "STEAM_1:0:95142811", version 2.
 
-; Changes from "GFL".
-; Fix "game_player_equip" dropping weapons and fix kevlar buying.
+// Delete low buy time, extend height of spawn trigger and fix zombies not getting teleported.
 modify:
 {
 	match:
 	{
-		"classname" "game_player_equip"
-		"targetname" "darP90"
+		"classname" "logic_auto"
+	}
+	insert:
+	{
+		"OnMapSpawn" "protect_player,AddOutput,solid 2,,1"
+		"OnMapSpawn" "protect_playerRunScriptCodeself.SetSize(Vector(-1600, -1178.5, -184.017), Vector(1600, 1178.5, 616));1"
+		"OnMapSpawn" "playerRunScriptCodeif (self.GetTeam() == 2 && self.GetHealth()) { self.SetOrigin(Vector(-12408, -8140, 318)); self.SetAngles(0, -89, 0); }301"
+	}
+	delete:
+	{
+		"OnMapSpawn" "consolaCommandmp_buytime 22-1"
+		"OnMapSpawn" "protect_playerEnable5-1"
+	}
+}
+
+// Is this needed? Not referenced in VMF or map vscript, only seems to be for spawning players but it isn't really needed.
```

Comment:
The fact it has a targetname like that is indeed interesting, but I checked this against v11_9 on CS:S and stripping player weapons does appear to be the intended functionality. So you should leave the change as is to maintain that.